docs/digger/installation/known-issues.asciidoc

```diff
@@ -1,52 +1,44 @@
 [[known-issues]]
```

Comment:
Was the known-issue section moved somewhere else?

Reply:
yes it was removed here: https://github.com/aerogear/aerogear.org/pull/700/files.
Is it to go somewhere else?

Reply:
or is it that it is only removed from the admin guide?

Reply:
sitrep: known-issues is in install guide
admin guide has an adoc "known-issues" also, but the only text there is an xref to ^ - does not make sense
todo: remove known-issues from admin guide, xref it from admin guide master directly.
idea: make known-issues a global doc - moving to level with "master" adocs. downside: same folder level will have master and not master docs. Workaround: new folder "shared"  @laurafitzgerald

Reply:
jiraed https://issues.jboss.org/browse/AGDIGGER-265